### PR TITLE
[#155215] Specify a stable version and use cdn.form.io

### PR DESCRIPTION
**Pull request by giladshanan**
As recommended by form.io support team

### CODE REVIEW

File: app/views/layouts/formio.html.haml [hunk 1]
```diff
@@ -3,9 +3,10 @@
   %head
     %meta{ "http-equiv" => "content-type", "content" => "text/html; charset=UTF-8" }
     = render_view_hook "head"
-    = stylesheet_link_tag "https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css", media: "all"
-    = stylesheet_link_tag "https://unpkg.com/formiojs@3.28.0/dist/formio.full.min.css", media: "all"
-    = javascript_include_tag "https://unpkg.com/formiojs@3.28.0/dist/formio.full.min.js"
+    - # using resource links based on https://github.com/formio/formio.js#form-building
```
Comment (boatrite) [inline app/views/layouts/formio.html.haml:6]: I think this technically should be `-#` for a haml comment

**Review by boatrite — APPROVED** [review]

File: app/views/layouts/formio.html.haml (hunk 1 above)
Comment (giladshanan) [reply app/views/layouts/formio.html.haml:6]: TIL 🎉

Comment (giladshanan) [reply app/views/layouts/formio.html.haml:6]: I wonder if HAML actually parses them differently

Comment (boatrite) [reply app/views/layouts/formio.html.haml:6]: Comments are likely stripped earlier, whereas non-haml-comments ruby comments have to go through the ruby parser, and so is presumably slower to get the final result

Comment (giladshanan) [inline app/views/layouts/formio.html.haml:6]:
```suggestion
    -# using resource links based on https://github.com/formio/formio.js#form-building
```

**Pull request merged**